services/api-gateway/Dockerfile

```
@@ -13,42 +13,69 @@
 
-# Set work directory
 WORKDIR /app
 
-# Copy requirements first for better caching
-COPY services/api-gateway/requirements.txt .
+COPY --from=uv /uv /uvx /usr/local/bin/
 
-# Install Python dependencies
-RUN pip install --upgrade pip && \
-    pip install -r requirements.txt
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)